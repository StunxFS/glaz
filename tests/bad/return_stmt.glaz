fn x() {
  return true;
}

fn x2() i32 {
  return false;
}

fn main() {
}
