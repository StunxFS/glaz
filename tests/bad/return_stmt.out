tests/bad/return_stmt.glaz:2:4: error: unexpected non-void return value in void function
     | 
   2 |   return true;
     |   ^^^^^^^^^^^
tests/bad/return_stmt.glaz:6:4: error: expected type `i32`, found `bool`
     | 
   6 |   return false;
     |   ^^^^^^^^^^^^
     = note: in return of function `x2`
glaz error: could not compile `return_stmt`, aborting due to 2 previous errors.
